nengo/utils/compat.py

```diff
@@ -30,6 +32,13 @@ def ensure_bytes(s):
         assert isinstance(s, bytes)
         return s
 
+    def replace(src, dst):
+        try:
+            os.rename(src, dst)
+        except OSError:  # Will be raised on Windows if dst exists
+            os.unlink(dst)
```

Comment:
The [docs say it's identical](https://docs.python.org/2/library/os.html#os.remove).